Sources/RoutingKit/Parameters.swift

```diff
@@ -18,33 +18,46 @@ public struct Parameters {
         values = [:]
     }
 
-    /// Grabs the named parameter from the parameter bag.
+    /// Grabs the parameter, supplied as a `PathComponent.parameter`, from the parameter bag.
     ///
     /// For example GET /posts/:post_id/comments/:comment_id
     /// would be fetched using:
     ///
-    ///     let postID = parameters.get("post_id")
-    ///     let commentID = parameters.get("comment_id")
+    ///     let postID = parameters.get(.postId)
+    ///     let commentID = parameters.get(.commentId)
     ///
-    public func get(_ name: String) -> String? {
-        return self.values[name]
+    public func get(_ parameter: PathComponent) -> String? {
```

Comment:
Is this a breaking change? If people are using the status quo `get(_:String)`, will their builds break?

Reply:
No it's not, the public API doesn't change because it doesn't use a named parameter